expyfun/stimuli/tests/test_tracker.py

```diff
@@ -164,12 +163,11 @@ def test_tracker_dealer():
     assert_raises(RuntimeError, dealer_ud.get_trial)
 
     # test array-like indexing
-    dealer_ud[0]
-    dealer_ud[:]
-    dealer_ud[[1, 0, 1]]
-    [d for d in dealer_ud]
+    dealer_ud.trackers[0]
+    dealer_ud.trackers[:]
+    dealer_ud.trackers[[1, 0, 1]]
+    [d for d in dealer_ud.trackers]
     dealer_ud.shape
```

Comment:
If it's a question of keep it or toss it, I'd say keep it. As long as there's nothing else we'd like to know the shape of.